README.md

```diff
@@ -19,7 +19,7 @@ A project that will render your data models into [JSONAPI Documents](http://json
 
 ## Documentation
 
-[Full docs here](https://hexdocs.pm/jsonapi/api-reference.html)
+[Full docs here](https://hexdocs.pm/jsonapi)
```

Comment:
Should we include a link to the actual JSONAPI spec here?

Reply:
Done!